gdal/ogr/ogrsf_frmts/geojson/libjson/random_seed.c

```diff
@@ -253,6 +253,8 @@ static int get_dev_random_seed(int *seed)
 
 #ifdef WIN32
 
+#if !defined(WINAPI_FAMILY) || (WINAPI_FAMILY == WINAPI_FAMILY_DESKTOP_APP)
```

Comment:
This would be better to use RTC_WINDOWS_UNIVERSAL as we define that for UWP. This makes it easier to identify our RTC specific changes too